decompressed/gui_file/www/lang/de-de/webui-core.po

```diff
@@ -489,8 +489,8 @@ msgstr "CWMP"
 msgid "CWMP Settings"
 msgstr "CWMP Einstellungen"
 
-msgid "Can't recover dslam version."
-msgstr "Kann die DSlam Version nicht erkennen."
+msgid "Can't recover DSLAM version."
```

Comment:
Changes to msgid (case-sensitive) must be checked on sources file...
i.e. https://github.com/Ansuel/tch-nginx-gui/search?q=Can%27t+recover+dslam+version.&unscoped_q=Can%27t+recover+dslam+version.

Reply:
Got it, should be fixed now